src/drivers/Qt/ConsoleDebugger.h

```diff
@@ -114,6 +114,8 @@ class debuggerBookmarkManager_t
 		std::map <int, debuggerBookmark_t*>::iterator internal_iter;
 };
 
+enum UpdateType { UPDATE_NONE, UPDATE_ALL, UPDATE_NO_SCROLL };
```

Comment:
This UpdateType name in the global namespace is a little to generic for me. Can we move this inside the QAsmView class public scope?

Reply:
Sure. I pushed that change.